autoload/maktaba/syscall.vim

```diff
@@ -216,6 +284,31 @@ endfunction
 
 ""
 " @dict Syscall
+" Executes the system asynchronously and invokes the callback on completion.
+" {callback} function will be called on asynchronous command completion, with
+" the following arguments: {callback}(env_dict, result_dict), where env_dict
+" contains tab, buffer, path, column and line info, and the result_dict contains
+" stdout, stderr and status (code).
+" If {allow_sync_fallback} is 1 and async calls are not available, a synchronous
+" call will be executed and callback called with the result.
+" If [throw_errors] is 1, any exit code from the command will cause a ShellError
+" to be thrown. Otherwise, the caller is responsible for checking
+" |v:shell_error| and handling error conditions.
```

Comment:
I was not even setting it to the dict, always setting 0 in the case of sync fallback.